### DISCUSSION TITLE
bibi为什么外国视频翻译不了字幕阿

### DISCUSSION THREAD

**Discussion by YamashitaTadahide** [immersive-translate/immersive-translate#1895] [Help]
### 概述 | Summary

<img width="779" alt="c94c51372b09f844e9f3b4b1c602d36" src="https://github.com/user-attachments/assets/1dc764a1-c38b-40ac-b4c0-ef11eba81f8f">
就翻译个页面 视频就翻译不了？就youtu课翻译视频 其他ins有些也翻译不了 不是说都可以翻译吗 真的无语了

### 补充信息 | Additional information

能不能解决阿 为什么bibi外国视频不能翻译字幕出来 只翻译界面阿

**Comment by keyboard3** [comment]
只有站点有字幕选项才能翻，插件是翻译原字幕，不可能凭空生成字幕

**Reply by YamashitaTadahide** [reply]
唉 好吧 谢谢了兄弟
